code/modules/admin/verbs/mentorhelp.dm

```diff
@@ -0,0 +1,561 @@
+/client/var/mentorhelptimerid = 0	//a timer id for returning the mhelp verb
+/client/var/datum/mentor_help/mentor_current_ticket	//the current ticket the (usually) not-mentor client is dealing with
+
+//
+//TICKET MANAGER
+//
+
+GLOBAL_DATUM_INIT(mhelp_tickets, /datum/mentor_help_tickets, new)
+
+/datum/mentor_help_tickets
```

Comment:
Problem is, the ticket stat counter fails to work with parenting admin and mentor. I tried my best to make it work but it doesn't work. If someone wants to try their hand at it, ping me on discord and i'll throw up what I have.